src/scry/completion/class_module_context.cr

```diff
@@ -0,0 +1,22 @@
+require "./context"
+
+module Scry::Completion
+  class ClassModuleContext < Context
+    def initialize(@text : String, @target : String, @method_db : Completion::MethodDB)
+    end
+
+    def find
+      target_header_index = @target.rindex(":")
+      @method_db.type_match(@target).map do |label|
+        label = label[(target_header_index + 1)..-1] if target_header_index
+        CompletionItem.new(
+          label: label,
+          insert_text: label,
+          kind: CompletionItemKind::Class,
+          detail: label,
```

Comment:
would it be better to have the full path (e.g: `JSON::Any`, not just `Any`) in details? (can someone test this, see how it goes?)

Reply:
you mean in the CompletionItem class ?